src/hooks/useDirty.ts

```diff
@@ -0,0 +1,15 @@
+import { useState, useMemo } from 'react';
+
+const useDirty = (chance?: number) => {
+  const [count, setCount] = useState(0);
+
+  const makeDirty = () => setCount((prev) => prev + 1);
+  const isDirty = useMemo(() => count > (chance || 0), [count, chance]);
```

Comment:
```suggestion
const useDirty = (chance?: number = 0) => {
  const [count, setCount] = useState(0);

  const makeDirty = () => setCount((prev) => prev + 1);
  const isDirty = useMemo(() => count > (chance), [count, chance]);
```
[의견]
이렇게 param 기본 값을 주는건 어떨까요?

Reply:
반영했습니다~  629b4db